gandalf.js

```diff
@@ -1,23 +1,22 @@
-const objectKeys = Object.keys(object);
```

Comment:
SCRIPT-Nice job on the comprehensive test coverage, it gives a lot of confidence in the stability of the new feature.